project/profiles/capacity4more/modules/c4m/og/c4m_og/plugins/entityreference/selection/C4MOgSelectionHandler.class.php

```diff
@@ -90,6 +90,13 @@ public function buildEntityFieldQuery(
     $user_groups = og_get_groups_by_user(NULL, $group_type);
     $user_groups = $user_groups ? $user_groups : array();
 
+    $current_gid = og_context();
```

Comment:
@ybaras 
`og_context()` returns the `$group` object, please rename the variable.